tools/docker/healthcheck.sh

```diff
@@ -4,7 +4,8 @@ HOST="localhost"
 PORT=$HEALTHCHECK_PORT
 
 if [ -z "$HEALTHCHECK_PORT" ]; then
-    PORT=$(netstat -tuln | grep -oE ':[0-9]+' | grep -oE '[0-9]+' | tail -n 1)
+    # check all the TCP listening sockets, filter the dragonfly process, and fetch the port
+    PORT=$(netstat -tlnp | grep "1/dragonfly" | grep -oE ':[0-9]+' | cut -c2-)
```

Comment:
It returns two ports if admin port enabled - 
```
6379
9999
```
Choosing one of them would fix it (piping it with `tail -n 1`)

Reply:
good catch!